sensors/solarwinds_api_check.yaml

```diff
@@ -0,0 +1,9 @@
+---
+  class_name: "SolarwindsApiCheck"
+  entry_point: "solarwinds_api_check.py"
+  description: "Verify that the SolarWinds API is up and accepting connections."
+  poll_interval: 60
```

Comment:
@jschoewe can you increase this polling interval to a higher-value, maybe 15 minutes?